x/ccv/provider/keeper/partial_set_security.go

```diff
@@ -97,11 +97,11 @@ func (k Keeper) OptInTopNValidators(ctx sdk.Context, chainID string, bondedValid
 }
 
 // ComputeMinPowerToOptIn returns the minimum power needed for a validator (from the bonded validators)
-// to belong to the `topN` validators. `chainID` is only used for logging purposes.
+// to belong to the `topN` validators for a Top N chain.
 func (k Keeper) ComputeMinPowerToOptIn(ctx sdk.Context, chainID string, bondedValidators []stakingtypes.Validator, topN uint32) (int64, error) {
-	if topN == 0 || topN > 100 {
-		return 0, fmt.Errorf("trying to compute minimum power with an incorrect topN value (%d)."+
-			"topN has to be between (0, 100]", topN)
+	if topN < 50 || topN > 100 {
+		return 0, fmt.Errorf("trying to compute minimum power for a Top N chain (%s) with an incorrect"+
+			" topN value (%d). topN has to be in [50, 100]", chainID, topN)
```

Comment:
should we make the lower/upper bounds constants somewhere, to keep things easier to maintain consistent if we ever need to update it?

Reply:
Good point. After the [above change](https://github.com/cosmos/interchain-security/pull/1925/files#r1627579199), `50` only shows up in a single location, in `(cccp *ConsumerAdditionProposal) ValidateBasic()` so I do not think it adds value to introduce a constant for it.